src/OrbitBase/LoggingUtilsTest.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "LoggingUtils.h"
+#include "absl/strings/str_format.h"
+#include "absl/time/clock.h"
+
+namespace {
+std::filesystem::path CreateTestLogFilePath(const absl::Time& timestamp) {
+  std::filesystem::path test_log_dir = "C:/OrbitAppDataDir/logs";
+  uint32_t test_pid = 12345;
+  std::string timestamp_string =
+      absl::FormatTime(kLogFileNameTimeFormat, timestamp, absl::LocalTimeZone());
+  std::string filename = absl::StrFormat(kLogFileNameDelimiter, timestamp_string, test_pid);
+  return test_log_dir / filename;
+}
+}  // namespace
+
+namespace orbit_base {
+
+TEST(LoggingUtils, GetLogFileLifetime) {
+  std::vector<std::string> test_cases({
+      "sfsdf-.log ",
+      "Orbitfoobar-2021_01_31T00_00_00-.log",
+      "Orbit-2021_01_31_00_00_00-7188.log",
+  });
```

Comment:
nitpick: Use variables instead of vector

const std::string kFilenameInvalidNoTimestamp = "sfsdf-.log ";
kFilenameInvalidValidTimestampNoPid
kFilenameValid

Reply:
Done. Thanks for the suggestion!